src/main/java/domain/LottoTicket.java

```diff
@@ -1,43 +1,37 @@
 package domain;
 
-import java.util.*;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 
 public class LottoTicket {
-    private static final String DUPLICATE_NUMBER_ERROR = "중복 숫자가 존재합니다.";
-    private static final String INCORRECT_LOTTO_NUMBER_SIZE_ERROR = "로또 숫자의 개수가 6이 아닙니다.";
+
     public static final int LOTTO_TICKET_SIZE = 6;
+    public static final int LOTTO_TICKET_PRICE = 1000;
+    private static final String INCORRECT_LOTTO_NUMBER_SIZE_ERROR = "로또 숫자의 개수가 6이 아닙니다.";
 
     private final Set<LottoNumber> lottoNumbers;
 
-    private LottoTicket(final Set<LottoNumber> lottoNumbers) {
-        validate(lottoNumbers);
+    private LottoTicket(final List<LottoNumber> numbers) {
+        LinkedHashSet<LottoNumber> lottoNumbers = new LinkedHashSet<>(numbers);
+        validateIncorrectSize(lottoNumbers);
```

Comment:
```suggestion
        Set<LottoNumber> lottoNumbers = new LinkedHashSet<>(numbers);
        validateIncorrectSize(lottoNumbers);
```
LinkedHashSet 으로 하신 이유가 따로 있을까요? 
Set으로 한다면 나중에 자료구조를 사용하게 되더라고 변경할 부분이 줄어들겠네요! 다형성도 함께 공부해보면 좋을 거 같아요 :)